packages/card/person-card/style.module.css

```diff
@@ -0,0 +1,35 @@
+.location {
+  composes: g-type-body-small-x-strong from global;
+  margin: 0;
+  color: var(--secondary-text-color);
```

Comment:
hmm, does this color get applied correctly even though `--secondary-text-color` is not defined in this file?

Reply:
Yep; verified that `--secondary-text-color` is successfully cascading down from `<Card />`:

![Slack_LlUdi8ihpw](https://user-images.githubusercontent.com/3746694/197853390-e16e70c6-4a4d-4932-9012-d0506523bf63.png)